pkgs/by-name/au/autoraise/package.nix

```diff
@@ -44,6 +44,7 @@ stdenv.mkDerivation rec {
     license = lib.licenses.gpl3Only;
     maintainers = with lib.maintainers; [ nickhu ];
     mainProgram = "autoraise";
-    platforms = lib.platforms.darwin;
+    # platforms = lib.platforms.darwin;
+    broken = stdenv.hostPlatform.isDarwin; # Bug: #355504
```

Comment:
Could you link to the actual issue or make note of why this needs to be marked broken instead of being fixed.

Reply:
Done. FYI, this package was using an explicitly private Apple API that no longer seems to be available.